js/sign/tests/cli_test.js

```diff
@@ -0,0 +1,32 @@
+import { runInThisContext } from 'vm';
+import * as cli from '../lib/cli.js';
+import * as wbnSign from '../lib/wbn-sign.js';
+
+const TEST_UNENCRYPTED_PRIVATE_KEY =
+  '-----BEGIN PRIVATE KEY-----\nMC4CAQAwBQYDK2VwBCIEIB8nP5PpWU7HiILHSfh5PYzb5GAcIfHZ+bw6tcd/LZXh\n-----END PRIVATE KEY-----';
+const TEST_ENCRYPTED_PRIVATE_KEY =
+  '-----BEGIN ENCRYPTED PRIVATE KEY-----\nMIGbMFcGCSqGSIb3DQEFDTBKMCkGCSqGSIb3DQEFDDAcBAhOw2E7LxOkzQICCAAw\nDAYIKoZIhvcNAgkFADAdBglghkgBZQMEASoEEJZqH2axMFEvdFmJLZlnch4EQMfJ\nAa/4uAmWqu2N5aOn2yIz3Ri+vQ/rzBPrvIaoDxYUUxwujJFujSbr3lnagHlOPptU\n7XhjbPbeOqidLqyv5rA=\n-----END ENCRYPTED PRIVATE KEY-----';
+
+function sleep(ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+describe('CLI key parsing', () => {
+  it('works for unencrypted key.', async () => {
+    const key = await cli.parseMaybeEncryptedKey(
+      Buffer.from(TEST_UNENCRYPTED_PRIVATE_KEY)
+    );
+    expect(key.type).toEqual('private');
+  });
+
+  it('works for unencrypted key read from `WEB_BUNDLE_SIGNING_PASSPHRASE`.', async () => {
+    process.env.WEB_BUNDLE_SIGNING_PASSPHRASE = 'helloworld';
+    const key = await cli.parseMaybeEncryptedKey(
+      Buffer.from(TEST_ENCRYPTED_PRIVATE_KEY)
+    );
+    expect(key.type).toEqual('private');
+    process.env.WEB_BUNDLE_SIGNING_PASSPHRASE = ''; // To not affect other tests.
```

Comment:
Could you put the reset of the env variable in finally block (`try {} finally {}`), or inside an [`afterEach`](https://jasmine.github.io/api/4.6/global.html#afterEach), so that it is definitely always called? 
Also, what do you think about using `delete process.env.WEB_BUNDLE_SIGNING_PASSPHRASE` instead of setting it to an empty string?